src/guide/extras/reactivity-in-depth.md

```diff
@@ -104,7 +104,7 @@ Code snippets here and below are meant to explain the core concepts in the simpl
 
 This explains a few [limitations of reactive objects](/guide/essentials/reactivity-fundamentals.html#limitations-of-reactive) that we have discussed in the fundamentals section:
 
-- When you assign or destructure a reactive object's property to a local variable, the reactivity is "disconnected" because access to the local variable no longer triggers the get / set proxy traps.
+- When you assign or destructure a reactive object's property to a local variable, the local variable's reactivity is "disconnected" from the parent object because access to the local variable no longer triggers the get / set proxy traps on the parent object.
```

Comment:
I'm not sure about this wording. I think it implies that the value of the property is also an object. The value could be anything. The phrase 'parent object' makes sense if there is also a child object, but I'm not so sure whether it works for values more generally.

Reply:
Thanks for the feedback

> The phrase 'parent object' makes sense if there is also a child object,  but I'm not so sure whether it works for values more generally.

I can't see it. I think the phrase makes sense if there is a parent object, regardless.

I'd agree, the gotcha here for me is all about when the child is an object. In my thinking if you destructure or assign a primitive value it seems obvious that the new variable will not be reactive -  how can it be. But the case of a ref type (array or object) is much more confusing because the thing is still a Proxy object, it's just disconnected.